src/components/pages/about-page/AboutPage.store.ts

```diff
@@ -0,0 +1,37 @@
+import { makeAutoObservable } from 'mobx';
+import { getErrorRequest } from '../../../domains/shows/shows.services';
+import { initialResponseStatus } from '../../../utils/mobx.utils';
+import { ApiResponse } from '../../../utils/http/http.types';
+import { getGlobalStore } from '../../shared/global-store-provider/GlobalStoreProvider';
+
+export class AboutPageStore {
+  globalStore = getGlobalStore();
+  errorExampleResults = initialResponseStatus<null>(null);
+
+  constructor() {
+    makeAutoObservable(this);
+  }
+
+  /**
+   * Store initializer. Should only be called once.
+   */
+  *init() {
+    yield Promise.all([this.loadSomething()]);
+  }
+
+  *loadSomething() {
+    const response: ApiResponse<null> = yield getErrorRequest();
+
+    this.errorExampleResults = {
+      data: this.errorExampleResults.data,
+      isRequesting: false,
+      ...response, // Overwrites the default data prop or adds an error. Also adds the statusCode.
+    };
+
+    if (response.error) {
+      const message = `${response.statusCode}: ${response.error.message}`;
+
+      this.globalStore.toastStore.enqueueToast(message, 'error');
+    }
+  }
+}
```

Comment:
@scottbromander from your experience in working with JavaScript developers do they not use Classes that often? I have found that is true for me and that is why I find myself working less and less with Classes. The other stores are just functions.